actions/interactWithPage.js

```diff
@@ -68,9 +69,14 @@ await page.getByText(articleByText, { exact: true }).click(articleByText);
 }
 
 async function execPlayWrightCode(page, code) {
+  const dependencies = [
+    {param: 'page', value: page},
+    {param: 'expect', value: expect},
+  ];
+
   try {
-    const func = AsyncFunction('page', code);
-    return await func(page);
+    const func = AsyncFunction(...dependencies.map((d) => d.param), code);
+    return await func(page, expect);
```

Comment:
Please update this line to use the dependencies too

Reply:
https://github.com/mayt/BrowserGPT/pull/17/commits/7c1fe6fc96dac077cfda26dc0bb1bd3d02ceb94b